src/utils/constants.ts

```diff
@@ -4,10 +4,9 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-
+import { CORE_EXTENSION_ID } from 'mobile-lsp-server';
```

Comment:
`SETTING_SECTION_DIAGNOSTIC` in line 13 is not used anywhere in the project. We can remove line 7, right?

Reply:
Good catch! our code becomes cleaner